newsroom/search.py

```diff
@@ -66,6 +66,13 @@ def get_filter_query(
     return {"terms": {aggregation_field: val}}
 
 
+class AdvancedSearchParams(TypedDict):
+    all: str
+    any: str
+    exclude: str
+    fields: str
```

Comment:
imo this might be still rather `List[str]`

Reply:
Whoops